nre/helm/indexer/values.writer.yaml

```diff
@@ -6,9 +6,9 @@ args: [
   "--db-url",
   "$(DB_URL)",
   "--rpc-client-url",
-  "https://public-rpc.mainnet.iota.io",
+  "https://api.mainnet.iota.cafe",
```

Comment:
There are still more without the port, just making sure you're aware :)

Reply:
Yes, the port is not really necessary for default https